pc/rpc_client.hpp

```diff
@@ -366,6 +366,19 @@ namespace pc
       node_map_t nmap_;
     };
 
+    // get current slot
+    class get_slot : public rpc_request
+    {
+    public:
+      get_slot( commitment = e_finalized );
```

Comment:
Should we default to `e_confirmed`? That seems to be the most common/useful slot measurement

Reply:
since this is a simple C++ wrapper over the Solana RPC api, I think the C++ default should match the Solana default